src/app/core/entity-components/entity-form/entity-form.service.ts

```diff
@@ -94,10 +94,11 @@ export class EntityFormService {
     this.extendFormFieldConfig(formFields, entity.getConstructor(), forTable);
     const formConfig = {};
     const entitySchema = entity.getSchema();
+    const copy = entity.copy();
```

Comment:
why do we copy here now? :thinking:

Reply:
When we are assigning the values to the form below, this was a shallow copy. This made the attendance being the same object for the actual entity and the one that is edited in the form. So pressing cancel did not properly reset the value. Therefore, we make sure to create a deep copy here.